File: Problem Solving/LeetCode/Minimum Operations to Make Array Equal.java
class Solution {
    public int minOperations(int n) {
        int ans = 0;
        int sum = 0;
        int arr[] = new int[n+1];
        for(int i=1; i<=n; i++) {
            arr[i] += 2 * i + 1;
            sum += arr[i];
        }
        int target = sum / n;
        
        for(int i=1; i<=n/2; i++) {
            ans += target - arr[i];
        }
        
        return ans;
    }
}

File: Problem Solving/LeetCode/Minimum Operations to Make Array Equal.go
package LeetCode

func minOperations(n int) int {
	ans := 0
	if n%2 == 0 {
		for i := 0; i < n/2; i++ {
			ans += (i * 2) + 1
		}
	} else {
		for i := 0; i < n/2; i++ {
			ans += ((n-1)/2)*2 + 1
			ans -= ((i * 2) + 1)
		}
	}
	return ans
}